runtime/lua/vim/lsp/client.lua

```diff
@@ -804,6 +804,7 @@ end
 ---
 --- @param force? boolean
 function Client:stop(force)
+  self.is_stopping = true
```

Comment:
also, if this new field is added, surly this method should check it before continuing?

Reply:
Right, maybe double stopping would not hurt, but its better for sure.